src/analytics/ValoraAnalytics.ts

```diff
@@ -109,6 +110,9 @@ class ValoraAnalytics {
         writeKey: SEGMENT_API_KEY,
         storePersistor: AsyncStoragePersistor,
       })
+      await LegacySegment.setup('n/a', {})
```

Comment:
nit: could be clearer what the n/a replaces

Reply:
```suggestion
      await LegacySegment.setup('n/a', {})  // n/a used in place of write key, which isn't needed because LegacySegment is only used for getAnonymousId
```

Reply:
btw did you test this without calling setup first? would be nice if we didn't need it

Reply:
I did test this without setup. doesn't work unfortunately :(